i18n/de.po

```diff
@@ -1,29 +1,42 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-#	* website_sale_osc
+# 	* website_sale_osc
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 8.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2015-05-11 14:28+0000\n"
-"PO-Revision-Date: 2015-05-11 14:28+0000\n"
+"POT-Creation-Date: 2015-05-26 14:56+0000\n"
+"PO-Revision-Date: 2015-05-26 17:01+0100\n"
 "Last-Translator: <>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: \n"
+"Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: \n"
+"Language: de\n"
+"X-Generator: Poedit 1.8\n"
 
 #. module: website_sale_osc
-#: view:website:website_sale_osc.osc_onestepcheckout
+#: view:website:website_sale_osc.address
 msgid "-- Create a new address --"
 msgstr "-- Neue Adresse anlegen --"
 
 #. module: website_sale_osc
-#: view:website:website_sale_osc.osc_onestepcheckout
+#: view:website:website_sale_osc.address
 msgid "Billing Information"
-msgstr "Persönliche Informationen"
+msgstr "Abrechnungsinformationen"
```

Comment:
We changed the translation of this string because with this long string we had troubles in the mobile view.